types/reach__router/index.d.ts

```diff
@@ -89,7 +91,10 @@ export interface MatchRenderProps<TParams> {
 
 export class Match<TParams> extends React.Component<MatchProps<TParams>> {}
 
-export type NavigateFn = (to: string, options?: NavigateOptions<{}>) => void;
+export interface NavigateFn {
+    (to: string, options?: NavigateOptions<{}>): Promise<void>;
+    (to: number): Promise<void>;
```

Comment:
could you point to the section in the docs showing where navigate can accept a number? does it also accept options in this case?

Reply:
Here: https://github.com/reach/router/blob/master/src/lib/history.js#L49-L51

Reply:
🤔 not sure if I like supporting undocumented features, but I guess this is fine

Reply:
By the way, there is a mention of this in the documentation :)
https://github.com/reach/router/blob/master/website/src/markdown/api/navigate.md#to